[Forms]:  Implement Size for Forms

ArtemKhvorostianyi opened the pull request:
<img width="1283" height="697" alt="Screenshot 2025-10-18 at 02 19 39" src="https://github.com/user-attachments/assets/9f9ba2b5-7c95-4ff0-86de-f6fdc5438109" />

rorychatt reviewed (changes requested):
Something is off with the checkbox.
ALso, we should increase the gap between elements. When the form is bigger, the spacing should also increase so the forms are more spatious.

See reference screenshot in original issue

ArtemKhvorostianyi commented:
Added different gaps, fixed checkboxes

<img width="1238" height="610" alt="Screenshot 2025-10-22 at 11 31 54" src="https://github.com/user-attachments/assets/8850bfca-c2b9-4f47-99cf-5d0db579d66a" />

Text at the top has the same size, because it's not actually a form part, it's a card's title, which contains form. Same thing for buttons, it isn't accessable to change its size (?).
<img width="1129" height="546" alt="Screenshot 2025-10-22 at 11 33 55" src="https://github.com/user-attachments/assets/ed977f67-7652-4b19-9718-6102722799a9" />

rorychatt commented:
@ArtemKhvorostianyi aha, I see. But then the Cards need support for Size.

But then I have this feedback:

<img width="477" height="658" alt="image" src="https://github.com/user-attachments/assets/52f5d331-2f48-452c-83cb-0c5803becef7" />

In forms space below a description needs to be smaller than above the description. So the description should be closer to the field it relates to, not closer to the one above it.

rorychatt commented on frontend/src/widgets/forms/FormWidget.tsx:
```diff
@@ -1,16 +1,34 @@
+      case Sizes.Large:
+        return 'gap-4 text-base [&_input:not([type="checkbox"])]:text-base [&_input:not([type="checkbox"])]:h-12 [&_input:not([type="checkbox"])]:px-4 [&_input:not([type="checkbox"])]:py-3 [&_textarea]:text-base [&_textarea]:min-h-[120px] [&_select]:text-base [&_select]:h-12 [&_label]:text-base [&_p]:text-base [&_button:not([role="checkbox"])]:text-base [&_button:not([role="checkbox"])]:h-12 [&_b ...
+      default:
+        return 'gap-3 text-sm [&_input:not([type="checkbox"])]:text-sm [&_input:not([type="checkbox"])]:h-10 [&_input:not([type="checkbox"])]:px-3 [&_input:not([type="checkbox"])]:py-2 [&_textarea]:text-sm [&_textarea]:min-h-[80px] [&_select]:text-sm [&_select]:h-10 [&_label]:text-sm [&_p]:text-sm [&_button:not([role="checkbox"])]:text-sm [&_button:not([role="checkbox"])]:h-10 [&_button:not([role ...
```
what is this css for?

rorychatt replied:
the paddings/gaps are BE defined... Maybe solution is to in the FormBuilder, when setting the Size, to override the paddings
